.github/workflows/test.yml

```diff
@@ -286,6 +286,7 @@ jobs:
         # are resolved, update this matrix list with '16'.
         # possibly fixed by https://github.com/jk1/Gradle-License-Report/pull/166
         java: [ '11' ]
+        gradle: ['current', '5.2']
```

Comment:
> Also, it looks like you're using some new properties in the init script, do you know how far back that new syntax is supported? Licensed generally aims to support all LTS versions of tools/frameworks so I want to make sure this change won't cause any backwards compatibility failures for users on older supported versions of gradle.

This is a very good point!
To simplify this task we could use the [gradle/gradle-build-action](https://github.com/gradle/gradle-build-action) to setup Gradle. 

We can then leverage the matrix to test different versions. 

This approach helps to keep track of gradle versions that are supported by licensed. 

The downside is that developers now need a local version of gradle to test  ([setup-source](https://github.com/gradle/gradle-build-action)), but this is also the case for other package managers.

Reply:
> The downside is that developers now need a local version of gradle to test ([setup-source](https://github.com/gradle/gradle-build-action)), but this is also the case for other package managers.

I think that would be a requirement for this PR anyway though, since the changes here get rid of the local `gradlew` executable.  As far as I know, the only reason there weren't previously checks around a local gradle installation was because that was included in the fixtures.  I might be wrong about that though, again I'm pretty low on contextual knowledge about gradle.